apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/index.php

```diff
@@ -184,7 +180,12 @@ function wpcom_track_global_styles( $blog_id, $post, $updated ) {
  * @return bool Returns true if custom styles are in use.
  */
 function wpcom_global_styles_in_use() {
-	$user_cpt = WP_Theme_JSON_Resolver_Gutenberg::get_user_data_from_wp_global_styles( wp_get_theme() );
+	if ( class_exists( 'WP_Theme_JSON_Resolver' ) ) {
+		$user_cpt = WP_Theme_JSON_Resolver::get_user_data_from_wp_global_styles();
```

Comment:
This fatally errors, the first ($theme) parameter is mandatory, even if you pass it null to get the current theme.

To repro, I created a site, added the sticker, sandboxed it and then visited the front-end of the site.

Reply:
After fixing this bug, the test steps work fine for me, however using 'Preview styles before upgrading' does nothing and the styles still aren't applied after upgrading. I assume that's the other existing caching issues rather than this though